dashboard/src/features/authentication/users/components/UsersBody/UsersBody.tsx

```diff
@@ -114,7 +113,9 @@ export default function UsersBody({ users, onSubmit }: UsersBodyProps) {
           phoneNumber: values.phoneNumber,
           phoneNumberVerified: values.phoneNumberVerified,
           locale: values.locale,
-          metadata,
+          ...(values?.metadata
+            ? { metadata: JSON.parse(values.metadata) }
+            : {}),
```

Comment:
I think the default value is an [empty string](https://github.com/nhost/nhost/pull/2689/files#diff-3f21e0e3338fdad19f473ea91876cc618fdf8e653da3673b30f49c2de21a087dR136), and in that case `values?.metadata` would be a falsy and so it will not be sent to the backend

Reply:
btw, just noticed an edge case that might have caused there which is:
if you create a user, add metadata, save and the come back to it to remove the metadata (basically clear the metadata field) it will not be sent to the server and so the metadata will not be reset.

Maybe we should consider sending `null` in that case

Reply:
Ah, ok! I'll fix